File: dart-jaguar-client/lib/model/edge_module_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/meta_edge_module_object.dart';
part 'edge_module_object.jser.dart';

class EdgeModuleObject {
  
  final String modulename;
  
  final String id;
  
  final String protocol;
  
  final String protocolprops;
  
  final MetaEdgeModuleObject meta;
  

  EdgeModuleObject(
    

{
    
     this.modulename = null,  
     this.id = null,  
     this.protocol = null,  
     this.protocolprops = null,  
     this.meta = null 
    }
  );

  @override
  String toString() {
    return 'EdgeModuleObject[modulename=$modulename, id=$id, protocol=$protocol, protocolprops=$protocolprops, meta=$meta, ]';
  }
}

@GenSerializer()
class EdgeModuleObjectSerializer extends Serializer<EdgeModuleObject> with _$EdgeModuleObjectSerializer {

}

File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/EdgeModuleObject.scala

package io.swagger.client.model

import java.util.HashMap

case class EdgeModuleObject (
    _modulename: String,
    __id: String,
    _protocol: String,
    _protocolprops: String,
    __meta: _metaEdgeModuleObject
)
object EdgeModuleObject {
    def toStringBody(var_modulename: Object, var__id: Object, var_protocol: Object, var_protocolprops: Object, var__meta: Object) =
        s"""
        | {
        | "modulename":$var_modulename,"_id":$var__id,"protocol":$var_protocol,"protocolprops":$var_protocolprops,"_meta":$var__meta
        | }
        """.stripMargin
}
